test/zebras/test_period.py

```diff
@@ -115,3 +115,40 @@ def test_periods_feature(freq):
     np.testing.assert_array_equal(pr.start_time.dayofweek, ps.dayofweek)
     np.testing.assert_array_equal(pr.start_time.dayofyear, ps.dayofyear)
     np.testing.assert_array_equal(pr.start_time.isocalendar().week, ps.week)
+
+
+@pytest.mark.parametrize(
+    "data, freq, result",
```

Comment:
Maybe

```suggestion
    "date, freq, result",
```

as well as below?